projects/boringssl/build.sh

```diff
@@ -18,8 +18,8 @@
 mkdir -p $WORK/boringssl
 cd $WORK/boringssl
 
-CFLAGS="$CFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE"
-CXXFLAGS="$CXXFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE"
+CFLAGS="$CFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE -D_BORINGSSL_LIBPKI_"
+CXXFLAGS="$CXXFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE -D_BORINGSSL_LIBPKI_"
```

Comment:
Nit: Maybe move `-D_BORINGSSL_LIBPKI_` this to line 42. Because this will get passed into both `cmake` and line 42, but `cmake` itself adds `-D_BORINGSSL_LIBPKI_`, so we'll end up building files twice.

Reply:
done